docs/src/appendixes/object_stores.md

```diff
@@ -1,4 +1,4 @@
-# Appendix A - Common object stores for backups
+# Common object stores for backups
```

Comment:
Why did you remove "Appendix A" @ebgitelman ?

Reply:
Appendixes are part of a book model. This is online content so the notion of an appendix doesn't really apply.